plugins/inputs/sqlserver/connectionstring.go

```diff
@@ -0,0 +1,124 @@
+package sqlserver
+
+import (
+	"net"
+	"net/url"
+	"strings"
+)
+
+const odbcPrefix = "odbc:"
+
+const (
+	emptyServerName   = "<empty-server-name>"
+	emptyDatabaseName = "<empty-database-name>"
+)
+
+// getConnectionIdentifiers returns the sqlInstance and databaseName from the given connection string.
+// The name of the server is returned as-is in the connection string
+// If the connection string could not be parsed or sqlInstance/databaseName were not present, a placeholder value is returned
+func getConnectionIdentifiers(connectionString string) (sqlInstance string, databaseName string) {
+	if len(connectionString) == 0 {
+		return
```

Comment:
Please use explicit returns.

Reply:
I've gone through all the `return`s and have attempted to clean them up to fix this Let me know if the current iteration still doesn't follow the guidelines.

Reply:
Looks good to me. Please close all comments of this type as resolved. Sorry I don't have the permission to do so...